### PR TITLE
DEV-2911 Default Country Codes to USA

### PR DESCRIPTION
**Pull request by BrianZito**
**Description:**
If transactions don't indicate any recipient country code, the country is assumed to be 'USA'. If transactions don't indicate any place of performance country code, the country is assumed to be 'USA'.

**Technical details:**
This makes no change in the data, only the matviews. In addition, country names will REMAIN NULL.
One of the docker-compose stages was split into two because usaspending-db-sql wasn't repeatable when reloading new .sql files (it was trying to repeat the `mkdir` command). README is updated.

**Requirements for PR merge:**

1. [N/A ] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
4. [x] Matview impact assessment completed
5. [ N/A] Frontend impact assessment completed
6. [ N/A] Data validation completed
7. [N/A ] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-2911](https://federal-spending-transparency.atlassian.net/browse/DEV-2911):
    - [x] Link to this Pull-Request

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

**Comment by BrianZito** [comment]
Would will be the right guy for that approval? If so I just got the go-ahead from him to make a different docker change, if not I'll just go ahead and take it all out since I was the last one using docker-compose

File: docker-compose.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ services:
     build: .
     volumes:
      - .:/dockermount
-    command: >
-      /bin/sh -c "mkdir $$MATVIEW_SQL_DIR &&
+    command:
+      /bin/sh -c "mkdir $$MATVIEW_SQL_DIR | true &&
```
Comment (emily-brents) [inline docker-compose.yaml]: Is this left over from the docker-compose stuff or did the changes to the matviews require this to be changed also?

Comment (BrianZito) [reply docker-compose.yaml]: This is a more compact fix to the original issue suggested to me by will. It is not strictly nessicary for the matview changes to work.

File: usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json [hunk 2]
```diff
@@ -40,16 +40,16 @@
     "  transaction_normalized.description AS transaction_description,",
     "  transaction_normalized.modification_number,",
     "",
-    "  COALESCE(transaction_fpds.place_of_perform_country_n, transaction_fabs.place_of_perform_country_n) AS pop_country_name,",
-    "  COALESCE(transaction_fpds.place_of_perform_country_c, transaction_fabs.place_of_perform_country_c) AS pop_country_code,",
+    "  COALESCE(transaction_fpds.place_of_perform_country_n, transaction_fabs.place_of_perform_country_n, 'USA') AS pop_country_name,",
```
Comment (kbard) [inline usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json]: Is this right?  Do we want to update name in addition to the code?

Comment (BrianZito) [reply usaspending_api/database_scripts/matview_generator/universal_transaction_matview.json]: Good catch; that was not intended behavior

**Pull request merged**